docs/querying/basics.md

```diff
@@ -4,17 +4,15 @@ nav_title: Basics
 sort_rank: 1
 ---
 
-# Querying Prometheus
+# Querying basics
 
 Prometheus provides a functional query language called PromQL (Prometheus Query
-Language) that lets the user select and aggregate time series data in real
+Language) that lets you select and aggregate time series data in real
 time. The result of an expression can either be shown as a graph, viewed as
 tabular data in Prometheus's expression browser, or consumed by external
 systems via the [HTTP API](api.md).
 
-## Examples
-
-This document is meant as a reference. For learning, it might be easier to
+> **Note:** This document is meant as a reference. For learning, it might be easier to
```

Comment:
No need to shout.

Reply:
This hasn't been reverted.

Reply:
I think this is worth calling out specifically, at least if we want to keep that strict separation.

Reply:
That applies to this entire section of docs though, not just this one.

Reply:
I agree, but isn't that aspect separate from how bold we make the markup around it?

Reply:
I see your point. This is already an entire section to itself, adding bold and a note on top of that seems excessive - the current text is already emphasised quite a bit.

Reply:
I removed the section titled Examples and moved the content to this note to reduce confusion. When a reader sees a section titled "Examples," they expect to have a section with examples in it. This was not what the section did. A note is clearer and more appropriate in this context.

Reply:
I can see that argument.